api/src/convert/mod.rs

```diff
@@ -35,34 +35,10 @@ mod tx_out_confirmation_number;
 mod tx_out_membership_element;
 mod tx_out_membership_proof;
 mod tx_prefix;
+mod verification_report;
 mod watcher;
 
-pub use account_key::*;
```

Comment:
Curious why those were removed. Do the `impl From/TryFrom` work without the use statements?

Reply:
Yep